### ISSUE TITLE
api.SharedWorker.error_event - Not firing in Chrome

### ISSUE THREAD

**Issue by haoma2514** [mdn/browser-compat-data#21385]
### What type of issue is this?

Browser bug (a bug with a feature that may impact site compatibility)

### What information was incorrect, unhelpful, or incomplete?

chrome version 119.0.6045.160
Error event cannot be triggered.

### What did you expect to see?

Error event is triggered.

### Did you test this? If so, how?

not triggered.

### Do you have anything more you want to share?

index.html script
```javascript
const worker2 = new SharedWorker("sharedWorker.js")
        console.log("worker2",worker2)
        worker2.port.start()
         
        worker2.port.addEventListener("message",(event)=>{
            console.log("received message",event)
        })
        btn2.addEventListener("click",()=>{            
            worker2.port.postMessage(input2.value)               
        })
        worker2.onerror =(x)=>{
            console.error("error",x)
            //will not be executed ，It's ok in firfox
        }
     
``` 
sharedWorker.js
```javascript
let x =0
var ports = [];
onconnect= function(e){
    const port = e.ports[0]
    ports.push(port)
    port.onmessage=(event)=>{ 
        x += parseInt(event.data)
        if(x>10)
            throw  "eee"
        port.postMessage(x)    
    }    
}
onerror=(x)=>{
    console.error("error",x)
}

self.onerror = function (e) { 
//Errors can be caught here.
    ports.forEach(function (port) { port.postMessage([e]); });
  };
``` 

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/SharedWorker/error_event

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `api.SharedWorker.error_event`
* Report started: 2023-11-29T06:00:46.341Z

</details>

**Comment by haoma2514** [comment]
Is automatically closed, do I need to post a comment?

**Comment by queengooborg** [comment]
We automatically close issues that contain the default title test as spammers often leave the title unchanged (or add a bit of spammy text to it).

It's hard to follow what's going on because your code is missing semicolons, so it has broken syntax.  Could you please fix the code and provide an easily reproducible example we can run in one click?

**Comment by ddbeck** [comment]
This bug report and test case isn't clear enough to act on, so I'm closing it. If this is still relevant, please file a new issue.
